pkg/ovs/ovsctl/appctl_test.go

```diff
@@ -15,13 +15,45 @@
 package ovsctl
 
 import (
+	"fmt"
+	"net"
+	"strings"
 	"testing"
 
+	gomock "github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	testDatapath1 = []byte(
+		`system@ovs-system:
+lookups: hit:37994604 missed:218759 lost:0
+flows: 5
+masks: hit:39862430 total:5 hit/pkt:1.04
+port 0: ovs-system (internal)
+port 1: vbr0 (internal)
+port 2: gre_sys (gre)
+port 3: net2`)
+	testDatapath2 = []byte(
+		`system@ovs-system:
+lookups: hit:37994604 missed:218759 lost:0
+flows: 5
+masks: hit:39862430 total:5 hit/pkt:1.04
+port 1: vbr0 (internal)
+port 2: gre_sys (gre)
+port 3: net2`)
+	testFlows = []string{
+		"table=3,ip,tcp,tcp_dst=80,action=output:2",
+		"table=2,ip,tcp,tcp_dst=22,action=output:1",
+		"table=0,in_port=3,ip,nw_src=1.1.1.2,action=resubmit(,2)",
+		"table=0,in_port=3,ip,nw_src=1.1.1.3,action=resubmit(,3)",
+	}
+)
+
 func TestOvsCtlClientGetDPFeatures(t *testing.T) {
```

Comment:
Given that the logic has been moved to `ovsctl.go`, tests in this function should be moved to `ovsct_test.go`. Same for `ofctl_test.go`.